### ISSUE TITLE
Fails for Xcode 7+ if used with a Development Prov Profile

### ISSUE THREAD

**Issue by viktorbenei** [bitrise-io/steps-xcode-archive#13]
Error:

```
set +v

(i) export_options_path: /Users/vagrant/deploy/export_options.plist
(i) archive_path: /Users/vagrant/deploy/THRPY.xcarchive

==> Collect infos from mobileprovision

==> Create export options

{:method=>"development"}
2015-10-28 09:09:19.433 xcodebuild[1820:8446] [MT] IDEDistribution: -[IDEDistributionLogging _createLoggingBundleAtPath:]: Created bundle at path '/var/folders/4f/945ywsh506g31f58fpcbg0840000gn/T/THRPY_2015-10-28_09-09-19.432.xcdistributionlogs'.
2015-10-28 09:09:19.813 xcodebuild[1820:8446] [MT] IDEDistribution: Step failed: <IDEDistributionSigningAssetsStep: 0x7fda54290b60>: Error Domain=IDEDistributionErrorDomain Code=1 "The operation couldnâ€™t be completed. (IDEDistributionErrorDomain error 1.)"
error: exportArchive: The operation couldnâ€™t be completed. (IDEDistributionErrorDomain error 1.)

Error Domain=IDEDistributionErrorDomain Code=1 "The operation couldnâ€™t be completed. (IDEDistributionErrorDomain error 1.)"

** EXPORT FAILED **
```

**Comment by viktorbenei** [comment]
We believe that the root cause of the issue is RVM, as the step works perfectly on OS X where RVM is not installed.

This StackOverflow link seems to confirm our hunch: http://stackoverflow.com/questions/33041109/xcodebuild-no-applicable-devices-found-when-exporting-archive

**Comment by gabosgab** [comment]
+1 Seeing the same issue.

**Comment by viktorbenei** [comment]
Fixed in v1.3.2 https://github.com/bitrise-io/steps-xcode-archive/releases/tag/1.3.2
